src/class-parsely.php

```diff
@@ -438,7 +438,13 @@ public function initialize_settings() {
 			)
 		);
 
-		$h = __( 'The plugin uses http canonical URLs by default. If this needs to be forced to use https, set this option to true. Note: the default is fine for almost all publishers, it\'s unlikely you\'ll have to change this unless directed to do so by a Parse.ly support rep.', 'wp-parsely' );
+		$parsely_options = $this->get_options();
+		if ( $parsely_options['force_https_canonicals'] === $this->is_home_url_with_https() ) {
+			$h = __( 'This option is being consistent with the protocol of your site address. It\'s unlikely you\'ll have to change this unless directed to do so by a Parse.ly support rep.', 'wp-parsely' );
+		} else {
+			$h = __( 'This option is not being consistent with the protocol of your site address. You can keep it as-is without affecting your Parse.ly data. However, if you\'re forced to change this value, please reach out to Parse.ly support', 'wp-parsely' );
```

Comment:
What about:

> Your site's default protocol appears to be https, but Parse.ly is currently crawling http links.  Don't worry, you can keep it as-is without affecting your Parse.ly data.  If you do decide to make a change, please reach out to a Parse.ly customer support representative so we can properly index your site after you've made this change.

Reply:
@mikeyarce Similar to above regrading the "consistent" word.

Also, IMHO, this change is a significant change to their data so I'd like to make sure they reach out to Parse.ly support first rather than let them change and reach out to Parse.ly support later. 

What do you think?